### PR TITLE
chore(ci): Fix brew cask install

### PR DESCRIPTION
**Pull request by kyletsang**
Fixes the following error

> Error: Calling brew cask install is disabled! Use brew install [--cask] instead.

### CODE REVIEW

**Review by jquense — APPROVED** [review]
feel free to just merge these sort of low risk CI fixes without a review @kyletsang 👍

**Comment by kyletsang** [comment]
> 
> 
> feel free to just merge these sort of low risk CI fixes without a review @kyletsang 👍

Seems like I don't have permission to do it without review.  The merge button is greyed out.

**Comment by kyletsang** [comment]
> 
> 
> > feel free to just merge these sort of low risk CI fixes without a review @kyletsang 👍
> 
> Seems like I don't have permission to do it without review. The merge button is greyed out.

@jquense 

https://github.community/t/how-to-allow-maintainers-to-merge-without-required-approvals/116601

Looks like ability to merge without review is limited to admins only.  Hopefully the permissions enhancements rolls out in the near future.

**Pull request merged**
